tensorrt_llm/_torch/modules/fla/layernorm_gated.py

```
@@ -0,0 +1,331 @@
+            NORM_BEFORE_GATE=norm_before_gate,
+            IS_RMS_NORM=is_rms_norm,
+            num_warps=num_warps,
+        )
+    return out, mean, rstd
+
+
+class LayerNormFn(torch.autograd.Function):
+
+    @staticmethod
+    def forward(
+        ctx,
```

Ruff (0.13.1)

Unused static method argument: `ctx`

(ARG004)